internal/runners/clean/run_lin_mac.go

```diff
@@ -166,7 +169,8 @@ func removeEmptyDir(dir string) error {
 	}
 
 	if !empty {
-		return errDirNotEmpty
+		fileList := strings.Join(fileutils.ListDirSimple(dir, true), "\n - ")
```

Comment:
If we have a templating mechanism for looping to avoid formatting here, please use that.

Reply:
Thanks. I didn't know that existed.

Reply:
My original intention was to avoid handling the view formatting here at all. However, the localization library itself is limited to string inputs, so there's no good way to do this at this time. This or the original solution is fine. Though, this centralizes the handling some and that might be best.